content/blog/getting-started-with-hpe-greenlake-api-for-virtualization.md

```diff
@@ -0,0 +1,412 @@
+                "type": "backup-recovery/data-orchestrator"
+            },
+            "hypervisorManagerType": "VMWARE_VCENTER",
+            "id": "213a7f0d-xxxx-xxxx-xxxx-44e597c32121",
+            "name": "rtp-array392-dhci.rtplab.nimblestorage.com",
+            "releaseVersion": "7.0.3",
+            "services": [
+                "hci-manager",
+                "backup-and-recovery"
+            ],
+            "state": "OK",
+            "status": "OK"
+        }
+    ],
+    "count": 2,
+    "offset": 0,
+    "total": 2
+}
+```
+
+*T﻿he above code snippet display the response from hyper-managers API execution*.
+
+### Creation and management of a virtual machine in a cloud service provider (AWS)
+
+At the time of this release  (March 2024), the API resource to discover the Cloud Service Provider (CSP) account is not yet available as part of this released HPE GreenLake API for virtualization. However, you can still obtain the information about the account Id at the CSP’s registration on HPE Private Cloud Business Enterprise’s Cloud account menu using the legacy CSP accounts list API `GET {baseUrl}/api/v1/csp-accounts`. Assuming that your AWS account had been onboarded into either HPE GreenLake Backup Recovery or HPE Private Cloud for Business Enterprise, this account Id will be used as one of the values for the body of this creation of virtual machine API as shown below.
+
+![](/img/list-available-csp-account-response.png "List available CSP accounts (DSCC API v1.4)")
+
+*T﻿he above figure shows the CSP id account that was onboarded into this workspace.*
+
+Another important information, the **imageId** value that is presented below, corresponds to the AWS Linux VM (free tier) that is going to be deployed at the AWS account. I used the following API `GET /virtualization/v1beta1/csp-machine-images` to discover that AWS machine image (AMI) from existing VM that had already been deployed inside the AWS Elastic Cloud Compute (ECS). You can use `GET /virtualization/v1beta1/csp-machine-instances` to get the machine-id required to execute the below API. 
+
+![](/img/obtain-machineid-from-a-machine-instance.png "return imageId from csp-machine-images")
+
+*T﻿he above figure shows the imageId value obtained from existing machine instance.*
+
+The figure below shows of the documentation of HPE GreenLake Virtualization API using a POST method to deploy a virtual machine in your CSP account: `POST /virtualization/v1beta1/csp-machine-instances`. Note that this API requires that you provide a **body** as part of the API invocation. The documentation for this virtualization API in the https://developer.greenlake.hpe.com uses another terminology which is called **Payload**.  It is presented as one of the tabs in the **Request samples** window, as shown below.  This **Payload** contains multiple key-pair values that is specified in the documentation for this API. 
```

Comment:
In this first sentence, you might be missing a word:
"The figure below shows [something] of the documentation..."

In the third sentence, change "terminology" to "term".

In the last sentence, change "that is specified" to "that are specified".